commerce-cloud-code/app_yourshophere/cartridge/partials/cart/checkoutbutton.js

```diff
@@ -14,7 +14,7 @@ exports.createModel = function createModel(product) {
     const currentBasket = BasketMgr.getCurrentBasket();
 
     const model = {
-        disabled: currentBasket == null || currentBasket.getTotalGrossPrice().value === 0,
+        disabled: currentBasket == null || !currentBasket.getTotalGrossPrice().available,
```

Comment:
This is good for priceless items or some tax setup problems, however I'd say || currentBasket.getTotalGrossPrice().value === 0 should be a third factor to prevent free orders

I only know one client where these are allowed, in all other cases the checkout and payment will get confused

Reply:
Mmh, I don't like that we don't allow it, if you want to give a 100% discount why not? Payment should ideally be skipped in this case and it would not be necessary

Reply:
80/20 ... In my opinion, the minority of merchants would need this functionality. Even if they offer free order, most would want to highly restrict entry for such orders to prevent mistakes. 
The chance is higher that someone slips through by accident.

But it's alright to not tackle this here. The better solution however would be to implement a configurable minimum order value, which we implement in almost every project

Reply:
I like that preference idea!

Reply:
https://github.com/your-shop-here/your-shop-here/issues/37